src/clj/puppetlabs/services/jruby/jruby_puppet_internal.clj

```diff
@@ -15,21 +15,19 @@
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;;; Definitions
 
-(def jruby-puppet-env
-  "The environment variables that should be passed to the Puppet JRuby interpreters.
-  We don't want them to read any ruby environment variables, like $GEM_HOME or
-  $RUBY_LIB or anything like that, so pass it an empty environment map - except -
-  Puppet needs HOME and PATH for facter resolution, so leave those."
-  (select-keys (System/getenv) ["HOME" "PATH"]))
-
 (def ruby-code-dir
-  "The name of the directory containing the ruby code in this project.
-  This directory lives under src/ruby/"
-  "puppet-server-lib")
+  "The name of the directory containing the ruby code in this project."
+  "src/ruby/puppet-server-lib")
```

Comment:
@camlow325 or @cprice404 or whomever feels confident about this...  Two questions I had while working with this bit of code:

 1. The path has been relative in the implementation up until this patch.  I'm not sure that's wise and/or secure for production since this is loading library code.  When my patch uses `ruby-code-dir` now, which is the only place it is used, the value is made absolute and canonical.  The question is: is this reasonable?  Is it a pre-mature improvement?
 2. Given that the path is relative and appears to be only intended to run things from source, does it make sense to include this path in the packaged releases of puppetserver?  I'm trying to come up with how this would be used in a production deployment and am falling short.